services/src/main/java/org/exoplatform/news/rest/NewsTargetingRestResourcesV1.java

```diff
@@ -79,4 +109,89 @@ public Response getReferencedTargets(@Context HttpServletRequest request) {
     }
   }
 
+  @DELETE
+  @Path("{targetName}")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @ApiOperation(value = "Delete target news", httpMethod = "DELETE", response = Response.class, notes = "This deletes the target news", consumes = "application/json")
+  @ApiResponses(value = { @ApiResponse(code = 200, message = "News target deleted"),
+          @ApiResponse(code = 400, message = "Invalid query input"),
+          @ApiResponse(code = 401, message = "User not authorized to delete the news target"),
+          @ApiResponse(code = 500, message = "Internal server error") })
+  public Response deleteTargetByName(@Context HttpServletRequest request,
+                                     @ApiParam(value = "Target name", required = true)
+                                     @PathParam("targetName") String targetName,
+                                     @ApiParam(value = "Time to effectively delete target news", required = false)
+                                     @QueryParam("delay") long delay) {
+    org.exoplatform.services.security.Identity currentIdentity = ConversationState.getCurrent().getIdentity();
+    try {
+      if (StringUtils.isBlank(targetName)) {
+        return Response.status(Response.Status.BAD_REQUEST).entity("Target name ist mandatory").build();
+      }
+      if (delay > 0) {
+        newsTargetToDeleteQueue.put(targetName, currentIdentity.getUserId());
+        scheduledExecutor.schedule(() -> {
+          if (newsTargetToDeleteQueue.containsKey(targetName)) {
+            ExoContainerContext.setCurrentContainer(container);
+            RequestLifeCycle.begin(container);
+            try {
+              newsTargetToDeleteQueue.remove(targetName);
+              newsTargetingService.deleteTargetByName(targetName, currentIdentity);
+            }  catch (Exception e) {
+              LOG.warn("Error when deleting the news target with name " + targetName, e);
+            } finally {
+              RequestLifeCycle.end();
+            }
+          }
+        }, delay, TimeUnit.SECONDS);
+      } else {
+        newsTargetToDeleteQueue.remove(targetName);
+        newsTargetingService.deleteTargetByName(targetName, currentIdentity);
+      }
+      return Response.ok().build();
+    } catch (Exception e) {
```

Comment:
you need to catch IllegalAccessException and return Response.Status.UNAUTHORIZED, see NewsRestResourcesV1 methods